core/src/test/java/org/jdbi/v3/core/TestBatchExceptionRewrite.java

```diff
@@ -66,8 +65,9 @@ public void testPreparedBatch() throws Exception
     private void assertSuppressions(Throwable cause) {
         LoggerFactory.getLogger(TestBatchExceptionRewrite.class).info("exception", cause);
         SQLException e = (SQLException) cause;
-        assertEquals(e.getNextException(), e.getSuppressed()[0]);
-        assertNull(e.getNextException().getNextException());
-        assertEquals(1, e.getSuppressed().length);
+        SQLException nextException = e.getNextException();
+        assertThat((Exception) nextException).isEqualTo(e.getSuppressed()[0]);
+        assertThat((Exception) nextException.getNextException()).isNull();
```

Comment:
Unfortunately not, because `SQLException` extends `Exception` and implement `Iterable` and the same time.